kf-portal-etl-docker/kf_etl.conf

```diff
@@ -41,6 +41,7 @@ io.kf.etl {
             mysql = ${io.kf.etl.mysql}
             mondo_path = "/kf-etl/mondo.tsv.gz"
             ncit_path = "/kf-etl/ncit.tsv.gz"
+            hpo_path = "/kf-etl/hpo_2018_03.tsv.gz"
```

Comment:
Would it be better to map to a version-generic path here, i.e. `hpo_path = "/kf-etl/hpo.tsv.gz"` and map the etl/volume to `"./kf-portal-etl-docker/hpo_2018_03.tsv.gz:/kf-etl/hpo.tsv.gz"` instead to avoid the duplication?

Reply:
Done